app/test/frontend/static_files_test.dart

```diff
@@ -12,8 +12,9 @@ import 'package:test/test.dart';
 
 import 'package:pub_dartlang_org/frontend/static_files.dart';
 
-Future main() async {
-  await updateLocalBuiltFiles();
+void main() {
+  test('init', () => updateLocalBuiltFiles(),
```

Comment:
Is this not supposed to be using a `setUp`, see: https://pub.dev/documentation/test_api/latest/test_api/setUp.html

Reply:
yeah... but there is no timeout for that. Not sure if this needs 10 minutes, I just remembered that in one of the projects I had the init time was more then the setup timeout... maybe I was remembering wrong and this was not that...